### ISSUE TITLE
lib: src: data: convert all data to JSON

### ISSUE THREAD

**Issue by yashlamba** [yashlamba/simulate#59]
As of now, we are using a lot of lists which aren't easy to keep a track of. Switching to json will make stuff easier as well as well structured.

**Comment by neil-dev** [comment]
I feel the same. Not only will the data be more structured but it will be easier to shift the data to an online storage, which might be needed when the app grows in size.

**Comment by yashlamba** [comment]
@neil-dev, I agree about it being structured though when I sat down to implement it, I didn't really see an absolute requirement for this. I would love to hear your thoughts.

**Comment by subhangi2731** [comment]
can you please assign this issue to me?

**Comment by sudhansukinng** [comment]
Hello @yashlamba, please assign this issue to me under GSSOC'21.

**Comment by yashlamba** [comment]
@subhangi2731 @sudhansukinng Thank you for your interest in this, we are not sure if this needs to be done or not rn. It's labelled discussion required, which means we still haven't finalized on this.
